src/ClassicUO.csproj

```diff
@@ -585,4 +586,4 @@
     <Folder Include="Renderer\UI\Layouts\" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Changes like this should not be committed. Please, adjust the settings of your editor to not add newlines automatically at the end of each file you hit save on. It should be done for the entire codebase at once or we will keep as it is even if it is ugly. :)

Reply:
Definitely agree, pulling now, running into issue with party atm.